README.md

```diff
@@ -202,6 +201,8 @@ This behavior is turned on by default, but configurable via the `--eigenda.confi
 
 In order to disperse to the EigenDA V1 network in production, or at high throughput on testnet, please register your authentication ethereum address through [this form](https://forms.gle/3QRNTYhSMacVFNcU8). Your EigenDA authentication keypair address should not be associated with any funds anywhere. For EigenDA V2, please see our [payments](https://docs.eigenda.xyz/releases/payments) doc.
 
+> Note: Proxy only supports using a single authn (v1) or payment (v2) key. We recommend running a single instance of the Proxy per Rollup.
```

Comment:
```suggestion
> Note: Proxy only supports using a single authorization (v1) or payment (v2) key. We recommend running a single instance of the Proxy per Rollup (i.e, 1 proxy : 1 rollup) and avoid provisioning patterns like N rollups : 1 proxy.
```

Reply:
Changed it to [505f8bd](https://github.com/Layr-Labs/eigenda-proxy/pull/353/commits/505f8bdde25ca138f3dd6b08fc9588f6ef3c078f)